whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/FullTextIndex.java

```diff
@@ -307,7 +307,31 @@ public RpslObject filterRpslObject(final RpslObject rpslObject) {
 
         return new RpslObject(rpslObject.getObjectId(), attributes);
     }
-    
+
+    public List<RpslAttribute> filterRpslAttributes(final Set<AttributeType> attributeType, final Map<String, Object> hitAttributes) {
+
+        List<RpslAttribute> attributes = Lists.newArrayList();
+
+        for (final AttributeType attribute : attributeType) {
+            if (SKIPPED_ATTRIBUTES.contains(attribute)) {
+                continue;
```

Comment:
Can you remove the SKIPPED_ATTRIBUTES before iterating from attributeType? Also this whole logic is already in ElasticAutocomplete Can you reuse it and put this in some helper class?

Reply:
the output in ElasticAutocomplete is a list of maps and here we have just a list. I could create a generic type and transforming that type but I think that it will make the logic more complicated

Reply:
about to move SKIPPED_ATTRIBUTES we need to iterate the list of attributes to skip those attributes that we don't want.